test/open.ts

```diff
@@ -41,6 +43,22 @@ suite('openDb', () => {
     assert.isTrue(upgradeRun, 'upgrade run');
   });
 
+  test('open without version - database never existed ', async () => {
+    db = (await openDB<TestDBSchema>(dbName)) as IDBPDatabase;
+
+    assert.strictEqual(db.version, 1);
+  });
+
+  test('open without version - database previously created', async () => {
+    const version = getNextVersion();
+    db = (await openDB<TestDBSchema>(dbName, version)) as IDBPDatabase;
+    db.close();
+
+    const db1 = (await openDB<TestDBSchema>(dbName)) as IDBPDatabase;
```

Comment:
Could this just reassign to `db`, so `db1` isn't needed?

Reply:
I actually did it on purpose, so if something fails on the way, we don't check the result of the first openDB() operation instead of the second. But if you think that is unnecessary, I will just change it to use the same variable :)